assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js

```diff
@@ -39,17 +39,17 @@ export default function SetupPublicationPendingVerification() {
 				'google-site-kit'
 			) }
 		>
-			<Link
+			<Button
+				className="mdc-button--surfaces"
```

Comment:
Let's use a new `inverse` prop for this instead, similar to how we have one for `danger`.

Reply:
Updated, thank you!